### ISSUE TITLE
Suggestion for reorganising exercises

### ISSUE THREAD

**Issue by anenadic** [carpentries/lesson-development-training#178]
Just recording some thoughts and a suggestion for reorganising exercises in the episodes: the Workbench, designing episode objectives and designing assessments:

- the exercise on creating new episode pages from the Workbench episode could be moved to the episode on designing episode objectives (where learners think about adding episodes in their lesson structure)
- the Workbench episode could have a small section on fenced divs and then an exercise to add prerequisites to the index page using the `prereq` fenced div to replace the exercise on creating episode pages (moved from the designing assessments episode)
- designing assessments episode could instead have an additional exercise to add the assessment/exercise for their lesson using fenced divs (that learners came up with as part of an earlier exercise in that episode). This can be in addition to the other existing exercise of adding their MCQ using fenced divs (this exercise can also be moved to the "stay on target" episode which mentioned MCQ).

**Comment by tobyhodges** [comment]
This one might be best tackled with #97, as it largely relates to the conversion of designed lesson content into Markdown with the Workbench.

**Comment by sstevens2** [comment]
I think these are great points to update.  I think #97 might take more discussion among the group but these seem might be able to be implemented in the meantime.

**Comment by anenadic** [comment]
Having thought about this issue and issue #97 (where @tobyhodges suggested we do lesson design on the first full day, and then Workbench, how we operate, and preparing to teach on the second full day) I propose to create a new issue to tackle both (even though #97 is closed).

**Comment by sstevens2** [comment]
@anenadic What do you think needs to be done once #380 is merged?

**Comment by anenadic** [comment]
> @anenadic What do you think needs to be done once #380 is merged?

Sorry, it was not clear from my above comment that I have created a separate issue #385 to propose the new structure for the course to be discussed in the trainers group.

**Comment by anenadic** [comment]
Closed via #385.
